### PR TITLE
WMS ID: #11358

### PR DESCRIPTION
**Pull request by ecchol**
#11358

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [Y] All filenames are lower case (including folders, images, files, etc.)
- [Y] Filenames are descriptive
- [Y ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [Y] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [N] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by anooshapilli — COMMENTED** [review]
@ecchol - There are a few file names with uppercase in them. Please make the markdown and image file name lowercase

File: ecc-hol/workshops/sandbox/manifest.json [hunk 1]
```diff
@@ -1,48 +1,68 @@
...
-        },
-        {
-            "title": "Need Help?",
-            "description": "Solutions to Common Problems and Directions for Receiving Live Help",
-            "filename":"https://oracle-livelabs.github.io/common/labs/need-help/need-help-livelabs.md"
-        }
+      {
+        "title": "Get started",
+        "description": "The Introduction is always second for LiveLabs. The title and contents menu title match for the Introduction.",
+        "filename": "../../query/Getstarted.md"
```
Comment (anooshapilli) [inline ecc-hol/workshops/sandbox/manifest.json:10]: @ecchol - Please use the common labs link -
Desktop version - https://oracle-livelabs.github.io/common//labs/remote-desktop/using-novnc-remote-desktop.md
 Sandbox version - https://oracle-livelabs.github.io/common//labs/cloud-login/cloud-login-livelabs2.md
Tenancy version - https://oracle-livelabs.github.io/common//labs/cloud-login/cloud-login.md

File: ecc-hol/workshops/sandbox/manifest.json [hunk 2]
```diff
@@ -1,48 +1,68 @@
...
+    "filename": "../../query/start-stop-ecc.md"
+  },
+  {
+    "title": "Additional optional labs: Improve performance",
+    "filename": "../../query/improve-performance.md"
+  },
+      {
+        "title": "Need help?",
+        "description": "Solutions to Common Problems and Directions for Receiving Live Help",
+        "filename": "../../query/needhelp.md"
```
Comment (anooshapilli) [inline ecc-hol/workshops/sandbox/manifest.json:64]: @ecchol Please use the common labs need help lab -

Desktop/Sandbox version - https://oracle-livelabs.github.io/common//labs/need-help/need-help-livelabs.md
Tenancy version - https://oracle-livelabs.github.io/common//labs/need-help/need-help-freetier.md

File: ecc-hol/workshops/sandbox/manifest.json (hunk 1 above)
Comment (anooshapilli) [reply ecc-hol/workshops/sandbox/manifest.json:10]: Also, delete the Get Started and Need Help lab in query folder.

**Review by anooshapilli — COMMENTED** [review]
@ecchol Create a folder for each of the labs and move the respective markdown file from the query folder into their lab-specific folder and update the file paths in manifest.json file

File: ecc-hol/workshops/sandbox/manifest.json (hunk 1 above)
Comment (ecchol) [reply ecc-hol/workshops/sandbox/manifest.json]: updated the file as per comments. Thank you

File: ecc-hol/workshops/sandbox/manifest.json (hunk 2 above)
Comment (ecchol) [reply ecc-hol/workshops/sandbox/manifest.json]: updated the file as per comments. Thank you

**Comment by anooshapilli** [comment]
Update on this PR: Had a call with Rahul. The workshop team will be making the requested changes to the workshop.

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**